marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mastering/default/merging.sjs

```diff
@@ -22,13 +22,24 @@ const processedURIs = [];
 
 function main(content, options) {
   // These index references can't be out this function scope or the jobReport will error, since they don't exist for the jobs DB
+  if (options.stepId) {
+    const stepDoc = fn.head(cts.search(cts.andQuery([
```

Comment:
Is it known what type of step this is - i.e. is it known to be a merging step, since this is the merging module? If so, can the artifacts/core library be used here instead to find the step document?